File: scripts/code2wiki/tests/fixtures/java/src/main/java/com/example/order/entity/OrderEntity.java
package com.example.order.entity;

import com.baomidou.mybatisplus.annotation.TableName;

@TableName("t_order")
public class OrderEntity {

    private Long id;
    private String orderNo;
    private OrderStatus status;
    private Long userId;
    private Long amount;

    public OrderStatus getStatus() { return status; }
    public void setStatus(OrderStatus s) { this.status = s; }
}

File: scripts/code2wiki/tests/fixtures/typescript/src/order/order.entity.ts
import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';

@Entity('t_order')
export class Order {
  @PrimaryGeneratedColumn()
  id: number;

  @Column()
  userId: number;

  @Column()
  amount: number;

  @Column()
  status: string;
}
